docs/DeveloperGuide.md

```diff
@@ -516,6 +516,8 @@ that this will be a heavy headway for our application.
 The following activity diagram summarizes what happens when a user executes a `focus` command.<br>
 <img src="images/FocusCommandDiagram.png" width="250" />
 
+
+#TO DELETE ****
```

Comment:
@charmainehly might need to add a spacing for the header and to appear in DG's TOC. Feel free to just commit and resolve via the button for convenience!

```suggestion
# TO DELETE ****
```